include/ubiquity_motor/motor_hardware.h

```diff
@@ -86,6 +88,14 @@ struct MotorDiagnostics {
     float battery_voltage_low_level = 22.5;
     float battery_voltage_critical = 21.0;
 
+    // Wheel current states
+    double motorCurrentLeft  = 0.0;
+    double motorCurrentRight = 0.0;
+    double motorAmpsPerAdcCount = 0.0238;   // 0.1V/Amp  2.44V=1024 count so 41.97 cnt/amp
```

Comment:
(optional) These are constants, they probably shouldn't be in this struct. If you chose to ignore this for now, please put a TODO comment here saying that it should be done.

Reply:
Half ok.   The amps per adc count will not change.
The zero offset is something I am seeing can be off.  Rather than work out a full calibration I wanted it to be able to be different.  We may do that cal if currents near zero are important enough to require.
So I will make one a constant but the zero offset will remain in the class.